src/api-binder.ts

```diff
@@ -79,7 +75,7 @@ export class APIBinder {
 	};
 	private eventTracker: EventTracker;
 
-	private balenaApi: PinejsClientRequest | null = null;
+	public balenaApi: PinejsClientRequest | null = null;
```

Comment:
can it be readonly if it's public?
I think ideally, it can be injected (through constructor) from outside...

Reply:
I don't believe that this can be marked `readonly`, as we don't initialize it in the constructor.